<div class="w-100-p" fxLayout="row" fxLayoutAlign="start start">

  <!-- Header -->
  <div class="header-container gradient">

    <!-- Static navbar -->
    <div class="container">
      <div class="header-nav">
        <div class="header-logo">
          <a class="pull-left" href="/">AnnoQ</a>
        </div>
        <div class="nav pull-right">
          <a routerLink="/search" target="_blank">Interactive Query</a>
          <a href="https://uscbiostats.github.io/AnnoQ/api" target="_blank">Data Access</a>
          <a href="/detail.html" target="_blank">Annotation Document</a>
          <a href="https://uscbiostats.github.io/AnnoQ/" target="_blank">About</a>
        </div>
      </div>
    </div>
    <!-- /Static navbar -->

    <!-- Header -->
    <!-- Header -->
    <div class="container header-wrapper">
      <div class="row">
        <div class="col-lg-12">
          <div class="header-content">
            <h1 class="header-title">AnnoQ: An integrated and interactive platform for large-scale genetic variant
              annotation.</h1>

            <div class="header-underline"></div>
            <div class="clearfix"></div>

          </div>
        </div>
      </div>
    </div>
    <!-- /Header -->
    <!-- /Header -->

  </div>
  <!-- /Header -->


  <!-- Content -->

  <div class="container content">
    <h1>Introduction</h1>
    <p>The Annotation Query (AnnoQ) is an integrated functional annotation platform for large scale genetic variant
      annotation. It serves researchers with different background and interests. Its easy-to-use user interface and
      search capabilities suit better for scientists with less bioinformatics skills, including bench scientists and
      statistician. The backend of the system is a large collection of pre-annotated variants from the Haplotype
      Reference Consortium (HRC) (~39 million) with sequence features (by WGSA) and functions (PANTHER and Gene
      Ontology) (we plan to add ~300 millions more variants from dbSNP in the future). The data is built in an
      Elasticsearch framework and an API was built to allow users to quickly access the annotation data, with one of the
      following three ways.
    </p>

    <div class="row">
      <div class="col-md-4">
        <div class="card mb-4 box-shadow">
          <a href="http://annoq.org:4206/">
            <img class="card-img-top" alt="GUI" style="" src="/static/main/img/gui.png" data-holder-rendered="true">
            <div class="centered item">Interactive Query</div>
          </a>
          <div class="card-body">
            <p class="card-text">Getting the annotation data with a graphic interface.</p>
            <div class="d-flex justify-content-between align-items-center">
              <div class="btn-group">
                <a href="http://annoq.org:4206/" type="button" class="btn btn-sm btn-outline-secondary">View</a>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div class="col-md-4">
        <div class="card mb-4 box-shadow">
          <a href="https://uscbiostats.github.io/AnnoQ/api">
            <img class="card-img-top" alt="API" style="" src="/static/main/img/api.png" data-holder-rendered="true">
            <div class="centered item">API Data Access</div>
          </a>
          <div class="card-body">
            <p class="card-text">Retrieving annotation data using scripts.</p>
            <div class="d-flex justify-content-between align-items-center">
              <div class="btn-group">
                <a href="https://uscbiostats.github.io/AnnoQ/api" type="button"
                  class="btn btn-sm btn-outline-secondary">View</a>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div class="col-md-4">
        <div class="card mb-4 box-shadow">
          <a href="/detail.html">
            <img class="card-img-top" alt="doc" style="height: 60%;width: 60%;display: block;"
              src="/static/main/img/document.png" data-holder-rendered="true">
            <div class="centered item">R Package</div>
          </a>
          <div class="card-body">
            <p class="card-text">Getting annotation data via R programming language.</p>
            <div class="d-flex justify-content-between align-items-center">
              <div class="btn-group">
                <a href="/detail.html" type="button" class="btn btn-sm btn-outline-secondary">View</a>
              </div>
            </div>
          </div>
        </div>
      </div>


    </div>
    <h1>GUI Tutorial</h1>
    <p>The UI for variants annotation query is available at <a href="http://annoq.org:4206/">http://annoq.org:4206/</a>
      .</p>
    <p>Here is a 20 second summary looping gif for tutorial. This can be handy to the editor or any user to know
      the UI. Also, I just quickly added to the UI Tutorial part below with bite sized sections. Kinda step by
      step</p>
    <p><img alt="short" src="/static/main/img/short.gif"></p>
    <p>The UI is composed of 3 main panels, the Query Panel, Result Panel and Result Summary Panel</p>
    <h2>1 Queries (Query Panel)</h2>
    <p>There 5 types of supported queries as described below (1-5). Choose a query type from the dropdown labeled
      “Change”.</p>
    <h3>1.1 Choose Query Type</h3>
    <p>Genome Coordinates Query: Chromosome location. Specify the chromosome number and the range. All data is
      based on hg19.
      Query by Upload VCF file: Variants List. Upload a VCF file of up to 10,000 variants by clicking the
      "Populate from a File" button. A sample VCF file can be found at here. Note that only first 50 rows of
      result will be displayed by our website. For retrieving the full result please using the download function.
      Gene Product Query: Gene Product. Input a gene or protein ID (e.g., ZMYND11). All variants located in the
      gene region will be returned. Currently, the system can only query one gene at a time. Gene name will be
      mapped to HGNC gene id by PANTHER gene mapping API.
      rsID query: Input a single rs ID and retrieve annotations.
      General Keyword Search: AnnoQ also supports full-text search by using a keyword. This keyword can be a gene
      name, a phenotype name or a GO term (e.g. Signaling by GPCR).</p>
    <h3>1.2 Select annotations</h3>
    <p>The annotations are organized in tree structure of categories and its individual annotations.</p>
    <p>Select the entire category by clicking on the checkbox (such as ANNOVAR).</p>
    <p>To choose individual annotations in a category, expand the tree by clicking the triangle next to the
      checkbox and click the desired checkboxes.</p>
    <h3>1.3 Submit query</h3>
    <p>Submit query by clicking the “Submit” button in the bottom of the panel. The results will be displayed on
      the result panel of the page.</p>
    <h3>1.4 Export configuration file</h3>
    <p>There are over 400 annotation types stored in our database. Users may not need all of them in their
      analysis. Through the process above, users can view the results and decide which annotation to use in their
      analysis. The configuration file stores the annotation selections the user chooses (from B above), and is
      used for command-line query or embedded in a programming script such as R. To generate a configuration
      file, simply click the “Export Config” button, and follow the instruction to save the file on the computer.
      For instructions how to use the configuration file in the command-line query or programming scripts, please
      visit link and AnnoQR. </p>
    <h2>2. Query Results (Results Panel)</h2>
    <p>The results are represented in a table format where each column ….</p>
    <p>To select an individual row for detailed result, click the row and the summary panel will appear</p>
    <h3>2.1 Download results</h3>
    <p>The results displayed on the results panel can be downloaded. To do so, click the “Download” button on the
      right upper corner of the results panel. The downloaded file is generated on the server. A link will be
      displayed. The user can click the link to download the file.</p>
    <p>It may take some time to generate the file for downloading.</p>
    <h2>3. Summary Panel</h2>
    <p>Select a row to see it in a detailed summary</p>



  </div>


  <!-- /Content -->

  <!-- Footer -->
  <div class="footer gradient-2">
    <div class="container footer-container ">
      <div class="row">
        <div class="col-xs-4 col-sm-3 col-md-3 col-lg-3">
          <div class="footer-title"><img src="/static/main/img/image_logo.gif" style="height:80px" alt="USC IMAGE logo">
          </div>
          <ul class="list-unstyled">
          </ul>
        </div>
        <div class="col-xs-4 col-sm-3 col-md-3 col-lg-3">

        </div>
        <div class="col-xs-4 col-sm-3 col-md-3 col-lg-3">
          <div class="footer-title"></div>
          <ul class="list-unstyled">
            <li><a href="http://annoq.org:4206/" target="_blank">Interactive Query</a></li>
            <li><a href="https://uscbiostats.github.io/AnnoQ/api" target="_blank">Data Access</a></li>
            <li><a href="/detail.html" target="_blank">Annotation Document</a></li>
            <li><a href="https://uscbiostats.github.io/AnnoQ/" target="_blank">About</a></li>
            <li><a href="mailto:annoqfeedback@gmail.com" target="_blank">Email US: annoqfeedback@gmail.com</a></li>
          </ul>
        </div>
        <div class="col-xs-12 col-sm-3 col-md-3 col-lg-3">
        </div>
      </div>
    </div>
  </div>
  <!-- /Footer -->

</div>