<div class="header-container gradient">

    <!-- Static navbar -->
    <div class="container">
        <div class="header-nav">
            <div class="header-logo">
                <a class="pull-left" href="/">AnnoQ</a>
            </div>
            <div class="nav pull-right">
                <a href="http://annoq.org:4206/" target="_blank">Interactive Query</a>
                <a href="https://uscbiostats.github.io/AnnoQ/api" target="_blank">Data Access</a>
                <a href="/detail.html" target="_blank">Annotation Document</a>
                <a href="https://uscbiostats.github.io/AnnoQ/" target="_blank">About</a>
            </div>
        </div>
    </div>
    <div class="container header-wrapper">
        <div class="row">
            <div class="col-lg-12">
                <div class="header-content">
                    <h1 class="header-title">AnnoQ: An integrated and interactive platform for large-scale genetic
                        variant annotation.</h1>

                    <div class="header-underline"></div>
                    <div class="clearfix"></div>

                </div>
            </div>
        </div>
    </div>
</div>
<div class="container">
    <div class="row">
        <div class="col">
            <h1>Annotation Details</h1>

            <dl class="row">
                <dt class="col-sm-4">chr</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">chromosome number</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">pos</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">position (hg19)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ref</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">reference allele</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">alt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">alternative allele</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ANNOVAR_ensembl_Effect</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">ANNOVAR consequence with Ensembl as gene model.</dd>
                <dd class="col-sm-8">Multiple consequences are separated by "|" The order corresponds to
                    ANNOVAR_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ANNOVAR_ensembl_Transcript_ID</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Ensembl transcript IDs corresponding to consequences.</dd>
                <dd class="col-sm-8">Multiple IDs are separated by "|"</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ANNOVAR_ensembl_Gene_ID</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Ensembl gene IDs corresponding to transcripts.</dd>
                <dd class="col-sm-8">Multiple IDs are separated by "|" The order corresponds to
                    ANNOVAR_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ANNOVAR_ensembl_Closest_gene(intergenic_only)</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">cloest genes if the variant is intergenic.</dd>
                <dd class="col-sm-8">Format gene ID1:transcript ID1(dist=distance),gene ID2:transcript
                    ID2(dist=distance)</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ANNOVAR_ensembl_HGVSc</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">variant in format of HGVSc.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    ANNOVAR_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ANNOVAR_ensembl_HGVSp</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">variant in format of HGVSp.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    ANNOVAR_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ANNOVAR_ensembl_Exon_Rank</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">exon number the variant locates (if applicable)</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    ANNOVAR_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ANNOVAR_ensembl_summary</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">ANNOVAR consequence summary with Ensembl as gene model.</dd>
                <dd class="col-sm-8">Format GeneID(total number of transcripts):consequence#1(number of transcripts
                    affected) consequence#2(number of transcripts affected)... Multiple genes are separated by "|"</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_ensembl_Effect</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">SnpEff consequence with Ensembl as gene model.</dd>
                <dd class="col-sm-8">Multiple consequences are separated by "|" The order corresponds to
                    SnpEff_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_ensembl_Effect_impact</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">consequence impacts predicted by SnpEff.</dd>
                <dd class="col-sm-8">Multiple impacts are separated by "|" The order corresponds to
                    SnpEff_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_ensembl_Sequence_feature</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">SnpEff consequence features.</dd>
                <dd class="col-sm-8">Multiple features are separated by "|" The order corresponds to
                    SnpEff_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_ensembl_Sequence_feature_impact</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">sequence feature impacts predicted by SnpEff.</dd>
                <dd class="col-sm-8">Multiple impacts are separated by "|" The order corresponds to
                    SnpEff_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_ensembl_Transcript_ID</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Ensembl transcript IDs corresponding to consequences.</dd>
                <dd class="col-sm-8">Multiple IDs are separated by "|"</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_ensembl_Transcript_biotype</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">biotype of transcript.</dd>
                <dd class="col-sm-8">Multiple biotypes are separated by "|" The order corresponds to
                    SnpEff_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_ensembl_Gene_name</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Ensembl gene names corresponding to transcripts.</dd>
                <dd class="col-sm-8">Multiple gene names are separated by "|" The order corresponds to
                    SnpEff_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_ensembl_Gene_ID</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Ensembl gene IDs corresponding to transcripts.</dd>
                <dd class="col-sm-8">Multiple IDs are separated by "|" The order corresponds to
                    SnpEff_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_ensembl_HGVSc</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">variant in format of HGVSc.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    SnpEff_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_ensembl_HGVSp</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">variant in format of HGVSp.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    SnpEff_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_ensembl_Protein_position/Protein_len</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">protein position and length as to a transcript.</dd>
                <dd class="col-sm-8">Format protein position/protein length Multiple annotations are separated by "|"
                    The order corresponds to SnpEff_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_ensembl_CDS_position/CDS_len</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">CDS position and length as to a transcript.</dd>
                <dd class="col-sm-8">Format CDS position/CDS length Multiple annotations are separated by "|" The order
                    corresponds to SnpEff_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_ensembl_cDNA_position/cDNA_len</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">cDNA position and length as to a transcript.</dd>
                <dd class="col-sm-8">Format cDNA position/cDNA length Multiple annotations are separated by "|" The
                    order corresponds to SnpEff_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_ensembl_Exon_or_intron_rank/total</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">exon (intron) rank and total number as to a transcript.</dd>
                <dd class="col-sm-8">Format exon or intron rank / total exon or intron number Multiple annotations are
                    separated by "|" The order corresponds to SnpEff_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_ensembl_Distance_to_feature</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">distance to feature.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    SnpEff_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_ensembl_Warnings</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">SnpEff warnings.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    SnpEff_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_ensembl_LOF/NMD</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">LOF or NMD by SnpEff.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_ensembl_LOF/NMD_gene_name</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">gene name corresponding to SnpEff_ensembl_LOF/NMD.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_ensembl_LOF/NMD_gene_ID</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">gene ID corresponding to SnpEff_ensembl_LOF/NMD.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_ensembl_LOF/NMD_num_transcripts_affected</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">number of transcripts affected</dd>
                <dd class="col-sm-8">corresponding to SnpEff_ensembl_LOF/NMD.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_ensembl_LOF/NMD_percent_transcripts_affected</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">percentage of transcripts affected</dd>
                <dd class="col-sm-8">corresponding to SnpEff_ensembl_LOF/NMD.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_ensembl_TF_binding_effect</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">effect of TF binding by SnpEff</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_ensembl_TF_name</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">name of the TF</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_ensembl_TF_ID</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">ID of the TF</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_ensembl_summary</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">SnpEff consequence summary with Ensembl as gene model.</dd>
                <dd class="col-sm-8">Format GeneID(total number of transcripts):consequence#1(number of transcripts
                    affected) consequence#2(number of transcripts affected)... Multiple genes are separated by "|"</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_ensembl_Consequence</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">VEP consequence with Ensembl as gene model.</dd>
                <dd class="col-sm-8">Multiple consequences are separated by "|" The order corresponds to
                    VEP_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_ensembl_Transcript_ID</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Ensembl transcript IDs corresponding to consequences.</dd>
                <dd class="col-sm-8">Multiple IDs are separated by "|"</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_ensembl_Gene_Name</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Ensembl gene names corresponding to transcripts.</dd>
                <dd class="col-sm-8">Multiple gene names are separated by "|" The order corresponds to
                    VEP_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_ensembl_Gene_ID</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Ensembl gene IDs corresponding to transcripts.</dd>
                <dd class="col-sm-8">Multiple IDs are separated by "|" The order corresponds to
                    VEP_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_ensembl_Protein_ID</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Ensembl protein IDs corresponding to transcripts.</dd>
                <dd class="col-sm-8">Multiple IDs are separated by "|" The order corresponds to
                    VEP_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_ensembl_CCDS</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">CCDS IDs corresponding to transcripts.</dd>
                <dd class="col-sm-8">Multiple IDs are separated by "|" The order corresponds to
                    VEP_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_ensembl_SWISSPROT</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">SWISSPROT IDs corresponding to transcripts.</dd>
                <dd class="col-sm-8">Multiple IDs are separated by "|" The order corresponds to
                    VEP_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_ensembl_Codon_Change_or_Distance</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">codon change or distance as to a transcript.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    VEP_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_ensembl_Amino_Acid_Change</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">amino acid change as to a transcript.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    VEP_ensembl_Transcript_ID "*" represents a stop codon</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_ensembl_HGVSc</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">variant in format of HGVSc.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    VEP_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_ensembl_HGVSp</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">variant in format of HGVSp.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    VEP_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_ensembl_cDNA_position</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">cDNA position as to a transcript.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    VEP_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_ensembl_CDS_position</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">CDS position as to a transcript.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    VEP_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_ensembl_Protein_position</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">protein position as to a transcript.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    VEP_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_ensembl_Exon_or_Intron_Rank</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">exon (intron) rank as to a transcript.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    VEP_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_ensembl_STRAND</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">strand of CDS as to a transcript.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    VEP_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_ensembl_CANONICAL</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">whether a transcript is canonical.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    VEP_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_ensembl_LoF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">LOF by LOFTEE.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    VEP_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_ensembl_LoF_filter</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">LOF filter by LOFTEE.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    VEP_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_ensembl_LoF_flags</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">LOF flags by LOFTEE.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    VEP_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_ensembl_LoF_info</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">LOF information by LOFTEE.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    VEP_ensembl_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_ensembl_summary</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">VEP consequence summary with Ensembl as gene model.</dd>
                <dd class="col-sm-8">Format GeneID(total number of transcripts):consequence#1(number of transcripts
                    affected) consequence#2(number of transcripts affected)... Multiple genes are separated by "|"</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ANNOVAR_refseq_Effect</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">ANNOVAR consequence with RefSeq as gene model.</dd>
                <dd class="col-sm-8">Multiple consequences are separated by "|" The order corresponds to
                    ANNOVAR_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ANNOVAR_refseq_Transcript_ID</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">RefSeq transcript IDs corresponding to consequences.</dd>
                <dd class="col-sm-8">Multiple IDs are separated by "|"</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ANNOVAR_refseq_Gene_ID</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">RefSeq gene IDs corresponding to transcripts.</dd>
                <dd class="col-sm-8">Multiple IDs are separated by "|" The order corresponds to
                    ANNOVAR_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ANNOVAR_refseq_Closest_gene(intergenic_only)</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">cloest genes if the variant is intergenic.</dd>
                <dd class="col-sm-8">Format gene ID1:transcript ID1(dist=distance),gene ID2:transcript
                    ID2(dist=distance)</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ANNOVAR_refseq_HGVSc</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">variant in format of HGVSc.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    ANNOVAR_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ANNOVAR_refseq_HGVSp</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">variant in format of HGVSp.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    ANNOVAR_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ANNOVAR_refseq_Exon_Rank</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">exon number the variant locates (if applicable)</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    ANNOVAR_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ANNOVAR_refseq_summary</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">ANNOVAR consequence summary with RefSeq as gene model.</dd>
                <dd class="col-sm-8">Format GeneID(total number of transcripts):consequence#1(number of transcripts
                    affected) consequence#2(number of transcripts affected)... Multiple genes are separated by "|"</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_refseq_Effect</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">SnpEff consequence with RefSeq as gene model.</dd>
                <dd class="col-sm-8">Multiple consequences are separated by "|" The order corresponds to
                    SnpEff_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_refseq_Effect_impact</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">consequence impacts predicted by SnpEff.</dd>
                <dd class="col-sm-8">Multiple impacts are separated by "|" The order corresponds to
                    SnpEff_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_refseq_Sequence_feature</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">SnpEff consequence features.</dd>
                <dd class="col-sm-8">Multiple features are separated by "|" The order corresponds to
                    SnpEff_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_refseq_Sequence_feature_impact</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">sequence feature impacts predicted by SnpEff.</dd>
                <dd class="col-sm-8">Multiple impacts are separated by "|" The order corresponds to
                    SnpEff_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_refseq_Transcript_ID</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">RefSeq transcript IDs corresponding to consequences.</dd>
                <dd class="col-sm-8">Multiple IDs are separated by "|"</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_refseq_Transcript_biotype</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">biotype of transcript.</dd>
                <dd class="col-sm-8">Multiple biotypes are separated by "|" The order corresponds to
                    SnpEff_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_refseq_Gene_name</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">RefSeq gene names corresponding to transcripts.</dd>
                <dd class="col-sm-8">Multiple gene names are separated by "|" The order corresponds to
                    SnpEff_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_refseq_Gene_ID</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">RefSeq gene IDs corresponding to transcripts.</dd>
                <dd class="col-sm-8">Multiple IDs are separated by "|" The order corresponds to
                    SnpEff_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_refseq_HGVSc</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">variant in format of HGVSc.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    SnpEff_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_refseq_HGVSp</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">variant in format of HGVSp.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    SnpEff_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_refseq_Protein_position/Protein_len</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">protein position and length as to a transcript.</dd>
                <dd class="col-sm-8">Format protein position/protein length Multiple annotations are separated by "|"
                    The order corresponds to SnpEff_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_refseq_CDS_position/CDS_len</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">CDS position and length as to a transcript.</dd>
                <dd class="col-sm-8">Format CDS position/CDS length Multiple annotations are separated by "|" The order
                    corresponds to SnpEff_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_refseq_cDNA_position/cDNA_len</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">cDNA position and length as to a transcript.</dd>
                <dd class="col-sm-8">Format cDNA position/cDNA length Multiple annotations are separated by "|" The
                    order corresponds to SnpEff_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_refseq_Exon_or_intron_rank/total</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">exon (intron) rank and total number as to a transcript.</dd>
                <dd class="col-sm-8">Format exon or intron rank / total exon or intron number Multiple annotations are
                    separated by "|" The order corresponds to SnpEff_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_refseq_Distance_to_feature</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">distance to feature.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    SnpEff_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_refseq_Warnings</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">SnpEff warnings.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    SnpEff_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_refseq_LOF/NMD</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">LOF or NMD by SnpEff.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_refseq_LOF/NMD_gene_name</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">gene name corresponding to SnpEff_refseq_LOF/NMD.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_refseq_LOF/NMD_gene_ID</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">gene ID corresponding to SnpEff_refseq_LOF/NMD.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_refseq_LOF/NMD_num_transcripts_affected</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">number of transcripts affected</dd>
                <dd class="col-sm-8">corresponding to SnpEff_refseq_LOF/NMD.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_refseq_LOF/NMD_percent_transcripts_affected</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">percentage of transcripts affected</dd>
                <dd class="col-sm-8">corresponding to SnpEff_refseq_LOF/NMD.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_refseq_TF_binding_effect</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">effect of TF binding by SnpEff</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_refseq_TF_name</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">name of the TF</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_refseq_TF_ID</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">ID of the TF</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SnpEff_refseq_summary</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">SnpEff consequence summary with RefSeq as gene model.</dd>
                <dd class="col-sm-8">Format GeneID(total number of transcripts):consequence#1(number of transcripts
                    affected) consequence#2(number of transcripts affected)... Multiple genes are separated by "|"</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_refseq_Consequence</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">VEP consequence with RefSeq as gene model.</dd>
                <dd class="col-sm-8">Multiple consequences are separated by "|" The order corresponds to
                    VEP_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_refseq_Transcript_ID</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">RefSeq transcript IDs corresponding to consequences.</dd>
                <dd class="col-sm-8">Multiple IDs are separated by "|"</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_refseq_Gene_Name</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">RefSeq gene names corresponding to transcripts.</dd>
                <dd class="col-sm-8">Multiple gene names are separated by "|" The order corresponds to
                    VEP_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_refseq_Gene_ID</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">RefSeq gene IDs corresponding to transcripts.</dd>
                <dd class="col-sm-8">Multiple IDs are separated by "|" The order corresponds to VEP_refseq_Transcript_ID
                </dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_refseq_Protein_ID</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">RefSeq protein IDs corresponding to transcripts.</dd>
                <dd class="col-sm-8">Multiple IDs are separated by "|" The order corresponds to VEP_refseq_Transcript_ID
                </dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_refseq_Codon_Change_or_Distance</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">codon change or distance as to a transcript.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    VEP_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_refseq_Amino_Acid_Change</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">amino acid change as to a transcript.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    VEP_refseq_Transcript_ID "*" represents a stop codon</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_refseq_HGVSc</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">variant in format of HGVSc.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    VEP_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_refseq_HGVSp</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">variant in format of HGVSp.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    VEP_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_refseq_cDNA_position</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">cDNA position as to a transcript.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    VEP_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_refseq_CDS_position</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">CDS position as to a transcript.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    VEP_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_refseq_Protein_position</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">protein position as to a transcript.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    VEP_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_refseq_Exon_or_Intron_Rank</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">exon (intron) rank as to a transcript.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    VEP_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_refseq_STRAND</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">strand of CDS as to a transcript.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    VEP_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_refseq_CANONICAL</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">whether a transcript is canonical.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    VEP_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_refseq_LoF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">LOF by LOFTEE.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    VEP_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_refseq_LoF_filter</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">LOF filter by LOFTEE.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    VEP_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_refseq_LoF_flags</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">LOF flags by LOFTEE.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    VEP_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_refseq_LoF_info</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">LOF information by LOFTEE.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    VEP_refseq_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_refseq_summary</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">VEP consequence summary with RefSeq as gene model.</dd>
                <dd class="col-sm-8">Format GeneID(total number of transcripts):consequence#1(number of transcripts
                    affected) consequence#2(number of transcripts affected)... Multiple genes are separated by "|"</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ANNOVAR_ucsc_Effect</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">ANNOVAR consequence with UCSC knownGene as gene model.</dd>
                <dd class="col-sm-8">Multiple consequences are separated by "|" The order corresponds to
                    ANNOVAR_ucsc_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ANNOVAR_ucsc_Transcript_ID</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">UCSC transcript IDs corresponding to consequences.</dd>
                <dd class="col-sm-8">Multiple IDs are separated by "|"</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ANNOVAR_ucsc_Gene_ID</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">UCSC gene IDs corresponding to transcripts.</dd>
                <dd class="col-sm-8">Multiple IDs are separated by "|" The order corresponds to
                    ANNOVAR_ucsc_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ANNOVAR_ucsc_Closest_gene(intergenic_only)</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">cloest genes if the variant is intergenic.</dd>
                <dd class="col-sm-8">Format gene ID1:transcript ID1(dist=distance),gene ID2:transcript
                    ID2(dist=distance)</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ANNOVAR_ucsc_HGVSc</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">variant in format of HGVSc.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    ANNOVAR_ucsc_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ANNOVAR_ucsc_HGVSp</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">variant in format of HGVSp.</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    ANNOVAR_ucsc_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ANNOVAR_ucsc_Exon_Rank</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">exon number the variant locates (if applicable)</dd>
                <dd class="col-sm-8">Multiple annotations are separated by "|" The order corresponds to
                    ANNOVAR_ucsc_Transcript_ID</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ANNOVAR_ucsc_summary</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">ANNOVAR consequence summary with UCSC as gene model.</dd>
                <dd class="col-sm-8">Format GeneID(total number of transcripts):consequence#1(number of transcripts
                    affected) consequence#2(number of transcripts affected)... Multiple genes are separated by "|"</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">rs_dbSNP151</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">rs number from dbSNP151</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">sno_miRNA_name</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">the name of snoRNA or miRNA if the site is located within (from miRBase/snoRNABase)
                </dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">sno_miRNA_type</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">the type of snoRNA or miRNA (from miRBase/snoRNABase)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">UTR3_miRNA_target</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">the gene-miRNA pair, if the site is located within a predicted (conserved)</dd>
                <dd class="col-sm-8">target of conserved miRNA families(from TargetScan)</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">TargetScan_context++_score_percentile</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">context++ score is a measure of favorableness of the</dd>
                <dd class="col-sm-8">site for the miRNA family. The higer the percentile, the more favorable (from
                    TargetScan)</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">splicing_consensus_ada_score</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">splicing-change prediction for splicing consensus SNPs</dd>
                <dd class="col-sm-8">based on adaboost. If the score >0.6, it predicts that the splicing will be
                    changed, otherwise it predicts the splicing will not be changed.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">splicing_consensus_rf_score</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">splicing-change prediction for splicing consensus SNPs</dd>
                <dd class="col-sm-8">based on random forest. If the score >0.6, it predicts that the splicing will be
                    changed, otherwise it predicts the splicing will not be changed</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">GWAS_catalog_rs</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">rs number according to GWAS catalog</dd>
                <dd class="col-sm-8">Please note the association is determined by rs number only, may refer to a
                    different allele</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">GWAS_catalog_trait</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">associated trait according to GWAS catalog.</dd>
                <dd class="col-sm-8">Please note the association is determined by rs number only, may refer to a
                    different allele</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">GWAS_catalog_pubmedid</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">pubmedid of the paper describing the association</dd>
                <dd class="col-sm-8">Please note the association is determined by rs number only, may refer to a
                    different allele</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">GWAS_catalog_OR</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">reported odds ratio associated with strongest risk allele</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">GRASP_rs</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">rs number by GRASP</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">GRASP_PMID</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">PMID number by GRASP</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">GRASP_p-value</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">p-value of the association test based on the SNP</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">GRASP_phenotype</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">phenotype the SNP associated with</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">GRASP_ancestry</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">population ancestry of the samples on which the association test was based</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">GRASP_platform</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">SNP platform on which the association test was based</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">clinvar_id</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">clinvar variation ID</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">clinvar_clnsig</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">clinical significance by clinvar</dd>
                <dd class="col-sm-8">Possible values Benign, Likely_benign, Likely_pathogenic, Pathogenic,
                    drug_response, histocompatibility. A negative score means the score is for the ref allele</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">clinvar_trait</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">the trait/disease the clinvar_clnsig referring to</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">clinvar_review</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">ClinVar Review Status summary</dd>
                <dd class="col-sm-8">Possible values no assertion criteria provided, criteria provided, single
                    submitter, criteria provided, multiple submitters, no conflicts, reviewed by expert panel, practice
                    guideline</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">clinvar_hgvs</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">variant in HGVS format</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">clinvar_var_source</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">source of the variant</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">clinvar_MedGen_id</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">MedGen ID of the trait/disease the clinvar_trait referring to</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">clinvar_OMIM_id</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">OMIM ID of the trait/disease the clinvar_trait referring to</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">clinvar_Orphanet_id</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Orphanet ID of the trait/disease the clinvar_trait referring to</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">COSMIC_ID</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">ID of the SNV at the COSMIC (Catalogue Of Somatic Mutations In Cancer) database
                </dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">COSMIC_CNT</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">number of samples having this SNV in the COSMIC database</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">GTEx_V7_gene</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">gene ID of the eQTL associated with</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">GTEx_V7_tissue</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">the tissue expression data in which the eQTL was detected</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">GERP_NR</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">GERP++ neutral rate</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">GERP_RS</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">GERP++ RS score, the larger the score, the more conserved the site</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">GERP_RS_rankscore</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">the rank of the GERP_RS score among all GERP_RS scores in genome</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">1000Gp3_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele counts in the whole 1000 genomes phase 3 (1000Gp3) data.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">1000Gp3_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the whole 1000Gp3 data.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">1000Gp3_AFR_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele counts in the 1000Gp3 African descendent samples.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">1000Gp3_AFR_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the 1000Gp3 African descendent samples.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">1000Gp3_EUR_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele counts in the 1000Gp3 European descendent samples.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">1000Gp3_EUR_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the 1000Gp3 European descendent samples.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">1000Gp3_AMR_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele counts in the 1000Gp3 American descendent samples.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">1000Gp3_AMR_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the 1000Gp3 American descendent samples.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">1000Gp3_EAS_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele counts in the 1000Gp3 East Asian descendent samples.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">1000Gp3_EAS_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the 1000Gp3 East Asian descendent samples.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">1000Gp3_SAS_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele counts in the 1000Gp3 South Asian descendent samples.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">1000Gp3_SAS_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the 1000Gp3 South Asian descendent samples.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">UK10K_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in called genotypes in UK10K cohorts.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">UK10K_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in called genotypes in UK10K cohorts.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">UK10K_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in called genotypes in UK10K cohorts.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">TWINSUK_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in called genotypes in UK10K TWINSUK cohort.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">TWINSUK_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in called genotypes in UK10K TWINSUK cohort.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">TWINSUK_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in called genotypes in UK10K TWINSUK cohort.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ALSPAC_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in called genotypes in UK10K TWINSUK cohort.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ALSPAC_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in called genotypes in UK10K TWINSUK cohort.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ALSPAC_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in called genotypes in UK10K TWINSUK cohort.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ESP6500_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele counts in the combined African/European American samples of the
                </dd>
                <dd class="col-sm-8">NHLBI GO Exome Sequencing Project (ESP6500 data set).</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ESP6500_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the combined African/European American samples of
                    the</dd>
                <dd class="col-sm-8">NHLBI GO Exome Sequencing Project (ESP6500 data set).</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ESP6500_AA_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele counts in the African American samples of the</dd>
                <dd class="col-sm-8">NHLBI GO Exome Sequencing Project (ESP6500 data set).</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ESP6500_AA_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the African American samples of the</dd>
                <dd class="col-sm-8">NHLBI GO Exome Sequencing Project (ESP6500 data set).</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ESP6500_EA_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele counts in the European American samples of the</dd>
                <dd class="col-sm-8">NHLBI GO Exome Sequencing Project (ESP6500 data set).</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ESP6500_EA_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the European American samples of the</dd>
                <dd class="col-sm-8">NHLBI GO Exome Sequencing Project (ESP6500 data set).</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ExAC_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Allele count in total ExAC samples (60,706 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ExAC_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Allele frequency in total ExAC samples</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ExAC_Adj_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Adjusted Alt allele counts (DP >= 10 & GQ >= 20) in total ExAC samples</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ExAC_Adj_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Adjusted Alt allele frequency (DP >= 10 & GQ >= 20) in total ExAC samples</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ExAC_AFR_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Adjusted Alt allele counts (DP >= 10 & GQ >= 20) in African & African American ExAC
                    samples</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ExAC_AFR_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Adjusted Alt allele frequency (DP >= 10 & GQ >= 20) in African & African American
                    ExAC samples</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ExAC_AMR_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Adjusted Alt allele counts (DP >= 10 & GQ >= 20) in American ExAC samples</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ExAC_AMR_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Adjusted Alt allele frequency (DP >= 10 & GQ >= 20) in American ExAC samples</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ExAC_EAS_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Adjusted Alt allele counts (DP >= 10 & GQ >= 20) in East Asian ExAC samples</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ExAC_EAS_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Adjusted Alt allele frequency (DP >= 10 & GQ >= 20) in East Asian ExAC samples</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ExAC_FIN_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Adjusted Alt allele counts (DP >= 10 & GQ >= 20) in Finnish ExAC samples</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ExAC_FIN_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Adjusted Alt allele frequency (DP >= 10 & GQ >= 20) in Finnish ExAC samples</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ExAC_NFE_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Adjusted Alt allele counts (DP >= 10 & GQ >= 20) in Non-Finnish European ExAC
                    samples</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ExAC_NFE_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Adjusted Alt allele frequency (DP >= 10 & GQ >= 20) in Non-Finnish European ExAC
                    samples</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ExAC_SAS_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Adjusted Alt allele counts (DP >= 10 & GQ >= 20) in South Asian ExAC samples</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ExAC_SAS_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Adjusted Alt allele frequency (DP >= 10 & GQ >= 20) in South Asian ExAC samples
                </dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_flag</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">information from gnomAD exome data indicating whether the variant falling within
                    low-complexity</dd>
                <dd class="col-sm-8">(lcr) or segmental duplication (segdup) or decoy regions. The flag can be either
                    "." for high-quality PASS or not reported/polymorphic in gnomAD exomes, "lcr" for within lcr,
                    "segdup" for within segdup, or "decoy" for with decoy region.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in the whole gnomAD exome samples (125,748 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in the whole gnomAD exome samples (125,748 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the whole gnomAD exome samples (125,748 samples)
                </dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of individuals with homozygous alternative allele in the whole gnomAD exome
                    samples (125,748 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_AFR_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in the African/African American gnomAD exome samples
                    (8,128 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_AFR_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in the African/African American gnomAD exome samples (8,128
                    samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_AFR_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the African/African American gnomAD exome samples
                    (8,128 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_AFR_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of individuals with homozygous alternative allele in the African/African
                    American gnomAD exome samples (8,128 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_AMR_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in the Latino gnomAD exome samples (17,296 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_AMR_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in the Latino gnomAD exome samples (17,296 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_AMR_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the Latino gnomAD exome samples (17,296 samples)
                </dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_AMR_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of individuals with homozygous alternative allele in the Latino gnomAD exome
                    samples (17,296 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_ASJ_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in the Ashkenazi Jewish gnomAD exome samples (5,040
                    samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_ASJ_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in the Ashkenazi Jewish gnomAD exome samples (5,040 samples)
                </dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_ASJ_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the Ashkenazi Jewish gnomAD exome samples (5,040
                    samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_ASJ_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of individuals with homozygous alternative allele in the Ashkenazi Jewish
                    gnomAD exome samples (5,040 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_EAS_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in the East Asian gnomAD exome samples (9,197 samples)
                </dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_EAS_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in the East Asian gnomAD exome samples (9,197 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_EAS_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the East Asian gnomAD exome samples (9,197 samples)
                </dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_EAS_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of individuals with homozygous alternative allele in the East Asian gnomAD
                    exome samples (9,197 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_FIN_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in the Finnish gnomAD exome samples (10,824 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_FIN_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in the Finnish gnomAD exome samples (10,824 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_FIN_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the Finnish gnomAD exome samples (10,824 samples)
                </dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_FIN_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of individuals with homozygous alternative allele in the Finnish gnomAD exome
                    samples (10,824 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_NFE_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in the Non-Finnish European gnomAD exome samples (56,885
                    samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_NFE_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in the Non-Finnish European gnomAD exome samples (56,885
                    samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_NFE_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the Non-Finnish European gnomAD exome samples
                    (56,885 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_NFE_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of individuals with homozygous alternative allele in the Non-Finnish European
                    gnomAD exome samples (56,885 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_SAS_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in the South Asian gnomAD exome samples (15,308 samples)
                </dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_SAS_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in the South Asian gnomAD exome samples (15,308 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_SAS_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the South Asian gnomAD exome samples (15,308
                    samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_SAS_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of individuals with homozygous alternative allele in the South Asian gnomAD
                    exome samples (15,308 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_POPMAX_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Allele count in the population with the maximum AF</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_POPMAX_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total number of alleles in the population with the maximum AF</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_POPMAX_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Maximum allele frequency across populations (excluding samples of Ashkenazi,
                    Finnish, and indeterminate ancestry)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_POPMAX_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of homozygous individuals in the population with the maximum allele frequency
                </dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in the controls subset of whole gnomAD exome samples
                    (54,704 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in the controls subset of whole gnomAD exome samples (54,704
                    samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the controls subset of whole gnomAD exome samples
                    (54,704 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of individuals with homozygous alternative allele in the controls subset of
                    whole gnomAD exome samples (54,704 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_AFR_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in the controls subset of African/African American gnomAD
                    exome samples (3,582 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_AFR_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in the controls subset of African/African American gnomAD exome
                    samples (3,582 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_AFR_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the controls subset of African/African American
                    gnomAD exome samples (3,582 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_AFR_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of individuals with homozygous alternative allele in the controls subset of
                    African/African American gnomAD exome samples (3,582 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_AMR_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in the controls subset of Latino gnomAD exome samples
                    (8,556 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_AMR_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in the controls subset of Latino gnomAD exome samples (8,556
                    samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_AMR_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the controls subset of Latino gnomAD exome samples
                    (8,556 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_AMR_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of individuals with homozygous alternative allele in the controls subset of
                    Latino gnomAD exome samples (8,556 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_ASJ_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in the controls subset of Ashkenazi Jewish gnomAD exome
                    samples (1,160 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_ASJ_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in the controls subset of Ashkenazi Jewish gnomAD exome samples
                    (1,160 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_ASJ_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the controls subset of Ashkenazi Jewish gnomAD
                    exome samples (1,160 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_ASJ_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of individuals with homozygous alternative allele in the controls subset of
                    Ashkenazi Jewish gnomAD exome samples (1,160 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_EAS_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in the controls subset of East Asian gnomAD exome samples
                    (4,523 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_EAS_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in the controls subset of East Asian gnomAD exome samples (4,523
                    samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_EAS_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the controls subset of East Asian gnomAD exome
                    samples (4,523 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_EAS_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of individuals with homozygous alternative allele in the controls subset of
                    East Asian gnomAD exome samples (4,523 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_FIN_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in the controls subset of Finnish gnomAD exome samples
                    (6,697 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_FIN_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in the controls subset of Finnish gnomAD exome samples (6,697
                    samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_FIN_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the controls subset of Finnish gnomAD exome samples
                    (6,697 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_FIN_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of individuals with homozygous alternative allele in the controls subset of
                    Finnish gnomAD exome samples (6,697 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_NFE_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in the controls subset of Non-Finnish European gnomAD
                    exome samples (21,384 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_NFE_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in the controls subset of Non-Finnish European gnomAD exome
                    samples (21,384 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_NFE_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the controls subset of Non-Finnish European gnomAD
                    exome samples (21,384 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_NFE_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of individuals with homozygous alternative allele in the controls subset of
                    Non-Finnish European gnomAD exome samples (21,384 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_SAS_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in the controls subset of South Asian gnomAD exome samples
                    (7,845 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_SAS_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in the controls subset of South Asian gnomAD exome samples
                    (7,845 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_SAS_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the controls subset of South Asian gnomAD exome
                    samples (7,845 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_SAS_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of individuals with homozygous alternative allele in the controls subset of
                    South Asian gnomAD exome samples (7,845 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_POPMAX_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Allele count in the controls subset of population with the maximum AF</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_POPMAX_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total number of alleles in the controls subset of population with the maximum AF
                </dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_POPMAX_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Maximum allele frequency across populations (excluding samples of Ashkenazi,
                    Finnish, and indeterminate ancestry) in the controls subset</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_exomes_controls_POPMAX_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of homozygous individuals in the controls subset of population with the
                    maximum allele frequency</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_flag</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">information from gnomAD genome data indicating whether the variant falling within
                    low-complexity</dd>
                <dd class="col-sm-8">(lcr) or segmental duplication (segdup) or decoy regions. The flag can be either
                    "." for high-quality PASS or not reported/polymorphic in gnomAD exomes, "lcr" for within lcr,
                    "segdup" for within segdup, or "decoy" for with decoy region.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in the whole gnomAD genome samples (15,708 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in the whole gnomAD genome samples (15,708 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the whole gnomAD genome samples (15,708 samples)
                </dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of individuals with homozygous alternative allele in the whole gnomAD genome
                    samples (15,708 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_AFR_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in the African/African American gnomAD genome samples
                    (4,359 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_AFR_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in the African/African American gnomAD genome samples (4,359
                    samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_AFR_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the African/African American gnomAD genome samples
                    (4,359 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_AFR_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of individuals with homozygous alternative allele in the African/African
                    American gnomAD genome samples (4,359 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_AMR_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in the Latino gnomAD genome samples (424 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_AMR_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in the Latino gnomAD genome samples (424 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_AMR_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the Latino gnomAD genome samples (424 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_AMR_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of individuals with homozygous alternative allele in the Latino gnomAD genome
                    samples (424 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_ASJ_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in the Ashkenazi Jewish gnomAD genome samples (145
                    samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_ASJ_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in the Ashkenazi Jewish gnomAD genome samples (145 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_ASJ_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the Ashkenazi Jewish gnomAD genome samples (145
                    samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_ASJ_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of individuals with homozygous alternative allele in the Ashkenazi Jewish
                    gnomAD genome samples (145 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_EAS_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in the East Asian gnomAD genome samples (780 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_EAS_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in the East Asian gnomAD genome samples (780 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_EAS_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the East Asian gnomAD genome samples (780 samples)
                </dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_EAS_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of individuals with homozygous alternative allele in the East Asian gnomAD
                    genome samples (780 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_FIN_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in the Finnish gnomAD genome samples (1,738 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_FIN_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in the Finnish gnomAD genome samples (1,738 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_FIN_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the Finnish gnomAD genome samples (1,738 samples)
                </dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_FIN_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of individuals with homozygous alternative allele in the Finnish gnomAD
                    genome samples (1,738 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_NFE_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in the Non-Finnish European gnomAD genome samples (7,718
                    samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_NFE_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in the Non-Finnish European gnomAD genome samples (7,718
                    samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_NFE_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the Non-Finnish European gnomAD genome samples
                    (7,718 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_NFE_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of individuals with homozygous alternative allele in the Non-Finnish European
                    gnomAD genome samples (7,718 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_POPMAX_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Allele count in the population with the maximum AF</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_POPMAX_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total number of alleles in the population with the maximum AF</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_POPMAX_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Maximum allele frequency across populations (excluding samples of Ashkenazi,
                    Finnish, and indeterminate ancestry)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_POPMAX_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of homozygous individuals in the population with the maximum allele frequency
                </dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in the controls subset of whole gnomAD genome samples
                    (5,442 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in the controls subset of whole gnomAD genome samples (5,442
                    samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the controls subset of whole gnomAD genome samples
                    (5,442 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of individuals with homozygous alternative allele in the controls subset of
                    whole gnomAD genome samples (5,442 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_AFR_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in the controls subset of African/African American gnomAD
                    genome samples (1,287 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_AFR_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in the controls subset of African/African American gnomAD genome
                    samples (1,287 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_AFR_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the controls subset of African/African American
                    gnomAD genome samples (1,287 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_AFR_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of individuals with homozygous alternative allele in the controls subset of
                    African/African American gnomAD genome samples (1,287 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_AMR_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in the controls subset of Latino gnomAD genome samples
                    (123 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_AMR_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in the controls subset of Latino gnomAD genome samples (123
                    samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_AMR_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the controls subset of Latino gnomAD genome samples
                    (123 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_AMR_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of individuals with homozygous alternative allele in the controls subset of
                    Latino gnomAD genome samples (123 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_ASJ_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in the controls subset of Ashkenazi Jewish gnomAD genome
                    samples (19 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_ASJ_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in the controls subset of Ashkenazi Jewish gnomAD genome samples
                    (19 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_ASJ_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the controls subset of Ashkenazi Jewish gnomAD
                    genome samples (145 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_ASJ_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of individuals with homozygous alternative allele in the controls subset of
                    Ashkenazi Jewish gnomAD genome samples (19 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_EAS_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in the controls subset of East Asian gnomAD genome samples
                    (458 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_EAS_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in the controls subset of East Asian gnomAD genome samples (458
                    samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_EAS_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the controls subset of East Asian gnomAD genome
                    samples (458 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_EAS_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of individuals with homozygous alternative allele in the controls subset of
                    East Asian gnomAD genome samples (458 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_FIN_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in the controls subset of Finnish gnomAD genome samples
                    (581 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_FIN_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in the controls subset of Finnish gnomAD genome samples (581
                    samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_FIN_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the controls subset of Finnish gnomAD genome
                    samples (581 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_FIN_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of individuals with homozygous alternative allele in the controls subset of
                    Finnish gnomAD genome samples (581 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_NFE_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele count in the controls subset of Non-Finnish European gnomAD
                    genome samples (2,762 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_NFE_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total allele count in the controls subset of Non-Finnish European gnomAD genome
                    samples (2,762 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_NFE_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Alternative allele frequency in the controls subset of Non-Finnish European gnomAD
                    genome samples (2,762 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_NFE_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of individuals with homozygous alternative allele in the controls subset of
                    Non-Finnish European gnomAD genome samples (2,762 samples)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_POPMAX_AC</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Allele count in the controls subset of population with the maximum AF</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_POPMAX_AN</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Total number of alleles in the controls subset of population with the maximum AF
                </dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_POPMAX_AF</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Maximum allele frequency across populations (excluding samples of Ashkenazi,
                    Finnish, and indeterminate ancestry) in the controls subset</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">gnomAD_genomes_controls_POPMAX_nhomalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Count of homozygous individuals in the controls subset of population with the
                    maximum allele frequency</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">RegulomeDB_motif</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">motif the SNP resides (from RegulomeDB)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">RegulomeDB_score</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">categorical score from RegulomeDB. The smaller, the more likely the SNP</dd>
                <dd class="col-sm-8">affects binding</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">Motif_breaking</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">whether break a known motif (in-house script)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">network_hub</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">whether the target gene is a network hub based on funseq-0.1</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ENCODE_annotated</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">whether annotated by ENCODE based on funseq-0.1</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">sensitive</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">whether defined as sensitive region based on funseq-0.1</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ultra_sensitive</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">whether defined as ultra-sensitive region based funseq-0.1</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">target_gene</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">target gene (for promoter, enhancer, etc.) based on funseq-0.1</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">funseq_noncoding_score</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">funseq-like noncoding score range 0-6, each of the previous 5 columns</dd>
                <dd class="col-sm-8">contribute 1 if "YES", or 0 if "NO"; the column Motif_breaking contribute 1 if it
                    is not a "."</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">CADD_raw</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">CADD raw score, the larger the number the more likely damaging</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">CADD_phred</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">CADD phred-like score, ranges 1-99, the larger the number the more likely</dd>
                <dd class="col-sm-8">damaging; score >10 means the variant in the top 10% (0.1) among the total 8.6
                    billion possible SNVs, >20 means in the top 1%, >30 means in the top 0.1%, etc. CADD suggests a
                    cutoff between 10 and 20 (e.g. 15)</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">CADD_raw_rankscore</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">the rank of the CADD_raw score among all CADD_raw scores in genome</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">fathmm-MKL_non-coding_score</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">fathmm-MKL non-coding prediction probability, the larger the number the more likely
                    damaging;</dd>
                <dd class="col-sm-8">the threshold separating deleterious prediction and neutral prediction is 0.5.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">fathmm-MKL_non-coding_rankscore</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">the rank of the fathmm-MKL_non-coding_score among all fathmm-MKL_non-coding_scores
                    in genome</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">fathmm-MKL_non-coding_pred</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">If a fathmm-MKL_non-coding_score is >0.5 the corresponding nsSNV</dd>
                <dd class="col-sm-8">is predicted as "D(AMAGING)"; otherwise it is predicted as "N(EUTRAL)".</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">fathmm-MKL_non-coding_group</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">fathmm-MKL non-coding group, the feature group used for the non-coding prediction
                </dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">fathmm-MKL_coding_score</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">fathmm-MKL coding prediction probability, the larger the number the more likely
                    damaging</dd>
                <dd class="col-sm-8">the threshold separating deleterious prediction and neutral prediction is 0.5.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">fathmm-MKL_coding_rankscore</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">the rank of the fathmm-MKL_coding_score among all fathmm-MKL_coding_scores in
                    genome</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">fathmm-MKL_coding_pred</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">If a fathmm-MKL_coding_score is >0.5 the corresponding nsSNV</dd>
                <dd class="col-sm-8">is predicted as "D(AMAGING)"; otherwise it is predicted as "N(EUTRAL)".</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">fathmm-MKL_coding_group</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">fathmm-MKL coding group, the feature group used for the coding prediction.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">fathmm-XF_score</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">fathmm-XF prediction probability, the larger the number the more likely damaging;
                </dd>
                <dd class="col-sm-8">the threshold separating deleterious prediction and neutral prediction is 0.5.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">fathmm-XF_rankscore</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">the rank of the fathmm-XF_score among all fathmm-XF_scores in genome</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">fathmm-XF_pred</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">If a fathmm-XF_score is >0.5 the corresponding nsSNV</dd>
                <dd class="col-sm-8">is predicted as "D(AMAGING)"; otherwise it is predicted as "N(EUTRAL)".</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">fathmm-XF_coding_or_noncoding</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">whether the prediction is based on the model for coding region</dd>
                <dd class="col-sm-8">or noncoding region</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ORegAnno_type</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">the type of regulatory region by ORegAnno</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ORegAnno_PMID</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">the PMID of the paper describing the regulation</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ENCODE_TFBS</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">name of the transcription factors (separated by ;) if the site is within a TFBS
                </dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ENCODE_TFBS_score</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">the higher the score the stronger the evidence of the TFBS</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ENCODE_TFBS_cells</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">the cell lines (separated by ;) the TFBS was detected</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ENCODE_Dnase_score</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">the higher the score the stronger the evidence of a DNase I</dd>
                <dd class="col-sm-8">hypersensitive site</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">ENCODE_Dnase_cells</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">number of cell lines supporting a DNase I hypersensitive site</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">FANTOM5_enhancer_permissive</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">whether the site is within a FANTOM5 phase 1+2 predicted permissive enhancer</dd>
                <dd class="col-sm-8">Y (Yes) or N (No)</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">FANTOM5_enhancer_robust</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">whether the site is within a FANTOM5 phase 1 predicted robust enhancer</dd>
                <dd class="col-sm-8">Y (Yes) or N (No)</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">FANTOM5_enhancer_target</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">if the site is within a FANTOM5 phase 1 predicted robust enhancer,</dd>
                <dd class="col-sm-8">this is the name of the FANTOM5 predicted target gene of the enhancer.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">FANTOM5_enhancer_expressed_tissue_cell</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">tissue or cell the enhancer expressed (from FANTOM5 phase 1).</dd>
                <dd class="col-sm-8">multiple values are separated by ; possible values
                    neuronal_stem_cell,myoblast,osteoblast,ciliated_epithelial_cell,
                    blood_vessel_endothelial_cell,mesothelial_cell,circulating_cell,T_cell,granulocyte,mast_cell,
                    sensory_epithelial_cell,astrocyte,mesenchymal_cell,fat_cell,chondrocyte,melanocyte,hepatocyte,
                    skeletal_muscle_cell,macrophage,keratinocyte,vascular_associated_smooth_muscle_cell,tendon_cell,
                    dendritic_cell,stromal_cell,neuron,reticulocyte,corneal_epithelial_cell,monocyte,acinar_cell,
                    natural_killer_cell,hepatic_stellate_cell,pericyte_cell,urothelial_cell,cardiac_myocyte,basophil,
                    neutrophil,lymphocyte_of_B_lineage,endothelial_cell_of_lymphatic_vessel,epithelial_cell_of_Malassez,
                    lens_epithelial_cell,epithelial_cell_of_prostate,epithelial_cell_of_esophagus,
                    mammary_epithelial_cell,preadipocyte,keratocyte,trabecular_meshwork_cell,respiratory_epithelial_cell,
                    enteric_smooth_muscle_cell,kidney_epithelial_cell,amniotic_epithelial_cell,cardiac_fibroblast,
                    fibroblast_of_choroid_plexus,fibroblast_of_the_conjuctiva,fibroblast_of_gingiva,
                    fibroblast_of_lymphatic_vessel,fibroblast_of_periodontium,fibroblast_of_pulmonary_artery,
                    hair_follicle_cell,intestinal_epithelial_cell,iris_pigment_epithelial_cell,placental_epithelial_cell,
                    retinal_pigment_epithelial_cell,bronchial_smooth_muscle_cell,smooth_muscle_cell_of_the_esophagus,
                    smooth_muscle_cell_of_trachea,uterine_smooth_muscle_cell,skin_fibroblast,gingival_epithelial_cell,
                    fibroblast_of_tunica_adventitia_of_artery,endothelial_cell_of_hepatic_sinusoid,
                    smooth_muscle_cell_of_prostate,lymph_node,large_intestine,blood,throat,testis,stomach,heart,brain,
                    eye,penis,female_gonad,uterus,vagina,adipose_tissue,esophagus,salivary_gland,skeletal_muscle_tissue,
                    smooth_muscle_tissue,urinary_bladder,pancreas,tongue,submandibular_gland,parotid_gland,blood_vessel,
                    placenta,thyroid_gland,lung,skin_of_body,spleen,liver,small_intestine,gallbladder,kidney,spinal_cord,
                    umbilical_cord,meninx,prostate_gland,thymus,tonsil,olfactory_region,internal_male_genitalia</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">FANTOM5_enhancer_differentially_expressed_tissue_cell</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">tissue or cell the enhancer expressed differentially</dd>
                <dd class="col-sm-8">(from FANTOM5 phase 1). multiple values are separated by ; possible values
                    neuronal_stem_cell,myoblast,osteoblast,ciliated_epithelial_cell,
                    blood_vessel_endothelial_cell,mesothelial_cell,circulating_cell,T_cell,granulocyte,mast_cell,
                    sensory_epithelial_cell,astrocyte,mesenchymal_cell,fat_cell,chondrocyte,melanocyte,hepatocyte,
                    skeletal_muscle_cell,macrophage,keratinocyte,vascular_associated_smooth_muscle_cell,tendon_cell,
                    dendritic_cell,stromal_cell,neuron,reticulocyte,corneal_epithelial_cell,monocyte,acinar_cell,
                    natural_killer_cell,hepatic_stellate_cell,pericyte_cell,urothelial_cell,cardiac_myocyte,basophil,
                    neutrophil,lymphocyte_of_B_lineage,endothelial_cell_of_lymphatic_vessel,epithelial_cell_of_Malassez,
                    lens_epithelial_cell,epithelial_cell_of_prostate,epithelial_cell_of_esophagus,
                    mammary_epithelial_cell,preadipocyte,keratocyte,trabecular_meshwork_cell,respiratory_epithelial_cell,
                    enteric_smooth_muscle_cell,kidney_epithelial_cell,amniotic_epithelial_cell,cardiac_fibroblast,
                    fibroblast_of_choroid_plexus,fibroblast_of_the_conjuctiva,fibroblast_of_gingiva,
                    fibroblast_of_lymphatic_vessel,fibroblast_of_periodontium,fibroblast_of_pulmonary_artery,
                    hair_follicle_cell,intestinal_epithelial_cell,iris_pigment_epithelial_cell,placental_epithelial_cell,
                    retinal_pigment_epithelial_cell,bronchial_smooth_muscle_cell,smooth_muscle_cell_of_the_esophagus,
                    smooth_muscle_cell_of_trachea,uterine_smooth_muscle_cell,skin_fibroblast,gingival_epithelial_cell,
                    fibroblast_of_tunica_adventitia_of_artery,endothelial_cell_of_hepatic_sinusoid,
                    smooth_muscle_cell_of_prostate,lymph_node,large_intestine,blood,throat,testis,stomach,heart,brain,
                    eye,penis,female_gonad,uterus,vagina,adipose_tissue,esophagus,salivary_gland,skeletal_muscle_tissue,
                    smooth_muscle_tissue,urinary_bladder,pancreas,tongue,submandibular_gland,parotid_gland,blood_vessel,
                    placenta,thyroid_gland,lung,skin_of_body,spleen,liver,small_intestine,gallbladder,kidney,spinal_cord,
                    umbilical_cord,meninx,prostate_gland,thymus,tonsil,olfactory_region,internal_male_genitalia</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">FANTOM5_CAGE_peak_permissive</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">whether the site is within a FANTOM5 phase 1 permissive Cap Analysis of Gene
                    Expression</dd>
                <dd class="col-sm-8">(CAGE) peak permissive set. Y (Yes) or N (No). A CAGE peak generally suggests a
                    promoter region</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">FANTOM5_CAGE_peak_robust</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">whether the site is within a FANTOM5 phase 1+2 robust Cap Analysis of Gene
                    Expression</dd>
                <dd class="col-sm-8">(CAGE) peak robust set. Y (Yes) or N (No). A CAGE peak generally suggests a
                    promoter region</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">Ensembl_Regulatory_Build_feature_type</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">genome segment prediction based on 68 cell types from</dd>
                <dd class="col-sm-8">ENCODE, BLUEPRINT and Roadmap. Predicted states CTCF_Binding_Site, Enhancer,
                    Open_chromatin, Promoter_Flanking_Region, TF_binding_site, Promoter</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">Ensembl_Regulatory_Build_ID</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Ensembl ID for the regulatory feature</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">Ensembl_Regulatory_Build_TFBS</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">TFBS from Ensembl Regulatory Build. Multiple TFBS separated by ";"</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">Ensembl_Regulatory_Build_TFBS_matrix</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">TF binding matrix ID</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">aaref</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">reference amino acid</dd>
                <dd class="col-sm-8">"." if the variant is a splicing site SNP (2bp on each end of an intron)</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">aaalt</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">alternative amino acid</dd>
                <dd class="col-sm-8">"." if the variant is a splicing site SNP (2bp on each end of an intron)</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">aapos</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">amino acid position as to the protein.</dd>
                <dd class="col-sm-8">"-1" if the variant is a splicing site SNP (2bp on each end of an intron). Multiple
                    entries separated by ";", corresponding to Ensembl_proteinid</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">genename</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">gene name; if the nsSNV can be assigned to multiple genes, gene names are</dd>
                <dd class="col-sm-8">separated by ";"</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">Ensembl_geneid</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Ensembl gene id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">Ensembl_transcriptid</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Ensembl transcript ids (Multiple entries separated by ";")</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">Ensembl_proteinid</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Ensembl protein ids</dd>
                <dd class="col-sm-8">Multiple entries separated by ";", corresponding to Ensembl_transcriptids</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">Uniprot_acc</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Uniprot accession number matching the Ensembl_proteinid</dd>
                <dd class="col-sm-8">Multiple entries separated by ";".</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">Uniprot_entry</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Uniprot entry ID matching the Ensembl_proteinid</dd>
                <dd class="col-sm-8">Multiple entries separated by ";".</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">APPRIS</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">APPRIS annotation for the transcripts matching Ensembl_transcriptid</dd>
                <dd class="col-sm-8">Multiple entries separated by ";". Potential values principal1, principal2,
                    principal3, principal4, principal5, alternative1, alternative2. See
                    https://useast.ensembl.org/info/genome/genebuild/transcript_quality_tags.html</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">GENCODE_basic</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Whether the transcript belongs to GENCODE_basic (5' and 3' complete</dd>
                <dd class="col-sm-8">transcripts). Multiple entries separated by ";", matching Ensembl_transcriptid. See
                    https://useast.ensembl.org/info/genome/genebuild/transcript_quality_tags.html</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">TSL</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Transcript Support Level.</dd>
                <dd class="col-sm-8">Multiple entries separated by ";", matching Ensembl_transcriptid. Potential values
                    1 to 5, NA. See https://useast.ensembl.org/info/genome/genebuild/transcript_quality_tags.html</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">VEP_canonical</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">canonical transcript used in Ensembl.</dd>
                <dd class="col-sm-8">Multiple entries separated by ";", matching Ensembl_transcriptid. See
                    https://useast.ensembl.org/Help/Glossary?id=521</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">cds_strand</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">coding sequence (CDS) strand (+ or -)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">refcodon</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">reference codon</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">codonpos</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">position on the codon (1, 2 or 3)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">codon_degeneracy</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">degenerate type (0, 2 or 3)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SIFT_score</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">SIFT score (SIFTori). Scores range from 0 to 1. The smaller the score the</dd>
                <dd class="col-sm-8">more likely the SNP has damaging effect. Multiple scores separated by ";",
                    corresponding to Ensembl_proteinid.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SIFT_converted_rankscore</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">SIFTori scores were first converted to SIFTnew=1-SIFTori,</dd>
                <dd class="col-sm-8">then ranked among all SIFTnew scores in dbNSFP. The rankscore is the ratio of the
                    rank the SIFTnew score over the total number of SIFTnew scores in dbNSFP. If there are multiple
                    scores, only the most damaging (largest) rankscore is presented. The rankscores range from 0.00964
                    to 0.91255.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SIFT_pred</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">If SIFTori is smaller than 0.05 (rankscore>0.39575) the corresponding nsSNV is</dd>
                <dd class="col-sm-8">predicted as "D(amaging)"; otherwise it is predicted as "T(olerated)". Multiple
                    predictions separated by ";"</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SIFT4G_score</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">SIFT 4G score (SIFT4G). Scores range from 0 to 1. The smaller the score the</dd>
                <dd class="col-sm-8">more likely the SNP has damaging effect. Multiple scores separated by ",",
                    corresponding to Ensembl_transcriptid</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SIFT4G_converted_rankscore</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">SIFT4G scores were first converted to SIFT4Gnew=1-SIFT4G,</dd>
                <dd class="col-sm-8">then ranked among all SIFT4Gnew scores in dbNSFP. The rankscore is the ratio of the
                    rank the SIFT4Gnew score over the total number of SIFT4Gnew scores in dbNSFP. If there are multiple
                    scores, only the most damaging (largest) rankscore is presented.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">SIFT4G_pred</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">If SIFT4G is < 0.05 the corresponding nsSNV is</dd>
                <dd class="col-sm-8">predicted as "D(amaging)"; otherwise it is predicted as "T(olerated)". Multiple
                    scores separated by ",", corresponding to Ensembl_transcriptid</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">LRT_score</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">The original LRT two-sided p-value (LRTori), ranges from 0 to 1.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">LRT_converted_rankscore</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">LRTori scores were first converted as LRTnew=1-LRTori*0.5 if</dd>
                <dd class="col-sm-8">Omega<1, or LRTnew=LRTori*0.5 if Omega>=1. Then LRTnew scores were ranked among all
                        LRTnew scores in dbNSFP. The rankscore is the ratio of the rank over the total number of the
                        scores in dbNSFP. The scores range from 0.00162 to 0.8433.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">LRT_pred</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">LRT prediction, D(eleterious), N(eutral) or U(nknown), which is not solely</dd>
                <dd class="col-sm-8">determined by the score.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">LRT_Omega</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">estimated nonsynonymous-to-synonymous-rate ratio (Omega, reported by LRT)</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">MutationTaster_score</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">MutationTaster p-value (MTori), ranges from 0 to 1.</dd>
                <dd class="col-sm-8">Multiple scores are separated by ";". Information on corresponding transcript(s)
                    can be found by querying http://www.mutationtaster.org/ChrPos.html</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">MutationTaster_converted_rankscore</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">The MTori scores were first converted. If the prediction</dd>
                <dd class="col-sm-8">is "A" or "D" MTnew=MTori; if the prediction is "N" or "P", MTnew=1-MTori. Then
                    MTnew scores were ranked among all MTnew scores in dbNSFP. If there are multiple scores of a SNV,
                    only the largest MTnew was used in ranking. The rankscore is the ratio of the rank of the score over
                    the total number of MTnew scores in dbNSFP. The scores range from 0.08979 to 0.81001.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">MutationTaster_pred</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">MutationTaster prediction, "A" ("disease_causing_automatic"),</dd>
                <dd class="col-sm-8">"D" ("disease_causing"), "N" ("polymorphism") or "P" ("polymorphism_automatic").
                    The score cutoff between "D" and "N" is 0.5 for MTnew and 0.31733 for the rankscore.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">MutationTaster_model</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">MutationTaster prediction models.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">MutationTaster_AAE</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">MutationTaster predicted amino acid change.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">MutationAssessor_score</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">MutationAssessor functional impact combined score (MAori). The</dd>
                <dd class="col-sm-8">score ranges from -5.17 to 6.49 in dbNSFP. Multiple entries are separated by ";",
                    corresponding to Uniprot_entry.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">MutationAssessor_score_rankscore</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">MAori scores were ranked among all MAori scores in dbNSFP.</dd>
                <dd class="col-sm-8">The rankscore is the ratio of the rank of the score over the total number of MAori
                    scores in dbNSFP. The scores range from 0 to 1.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">MutationAssessor_pred</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">MutationAssessor's functional impact of a variant -</dd>
                <dd class="col-sm-8">predicted functional, i.e. high ("H") or medium ("M"), or predicted non-functional,
                    i.e. low ("L") or neutral ("N"). The MAori score cutoffs between "H" and "M", "M" and "L", and "L"
                    and "N", are 3.5, 1.935 and 0.8, respectively. The rankscore cutoffs between "H" and "M", "M" and
                    "L", and "L" and "N", are 0.9307, 0.52043 and 0.19675, respectively.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">FATHMM_score</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">FATHMM default score (weighted for human inherited-disease mutations with</dd>
                <dd class="col-sm-8">Disease Ontology) (FATHMMori). Scores range from -16.13 to 10.64. The smaller the
                    score the more likely the SNP has damaging effect. Multiple scores separated by ";", corresponding
                    to Ensembl_proteinid.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">FATHMM_converted_rankscore</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">FATHMMori scores were first converted to</dd>
                <dd class="col-sm-8">FATHMMnew=1-(FATHMMori+16.13)/26.77, then ranked among all FATHMMnew scores in
                    dbNSFP. The rankscore is the ratio of the rank of the score over the total number of FATHMMnew
                    scores in dbNSFP. If there are multiple scores, only the most damaging (largest) rankscore is
                    presented. The scores range from 0 to 1.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">FATHMM_pred</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">If a FATHMMori score is <=-1.5 (or rankscore>=0.81332) the corresponding nsSNV</dd>
                <dd class="col-sm-8">is predicted as "D(AMAGING)"; otherwise it is predicted as "T(OLERATED)". Multiple
                    predictions separated by ";", corresponding to Ensembl_proteinid.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">PROVEAN_score</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">PROVEAN score (PROVEANori). Scores range from -14 to 14. The smaller the score</dd>
                <dd class="col-sm-8">the more likely the SNP has damaging effect. Multiple scores separated by ";",
                    corresponding to Ensembl_proteinid.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">PROVEAN_converted_rankscore</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">PROVEANori were first converted to PROVEANnew=1-(PROVEANori+14)/28,</dd>
                <dd class="col-sm-8">then ranked among all PROVEANnew scores in dbNSFP. The rankscore is the ratio of
                    the rank the PROVEANnew score over the total number of PROVEANnew scores in dbNSFP. If there are
                    multiple scores, only the most damaging (largest) rankscore is presented. The scores range from 0 to
                    1.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">PROVEAN_pred</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">If PROVEANori <= -2.5 (rankscore>=0.54382) the corresponding nsSNV is</dd>
                <dd class="col-sm-8">predicted as "D(amaging)"; otherwise it is predicted as "N(eutral)". Multiple
                    predictions separated by ";", corresponding to Ensembl_proteinid.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">MetaSVM_score</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Our support vector machine (SVM) based ensemble prediction score, which</dd>
                <dd class="col-sm-8">incorporated 10 scores (SIFT, PolyPhen-2 HDIV, PolyPhen-2 HVAR, GERP++,
                    MutationTaster, Mutation Assessor, FATHMM, LRT, SiPhy, PhyloP) and the maximum frequency observed in
                    the 1000 genomes populations. Larger value means the SNV is more likely to be damaging. Scores range
                    from -2 to 3 in dbNSFP.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">MetaSVM_rankscore</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">MetaSVM scores were ranked among all MetaSVM scores in dbNSFP.</dd>
                <dd class="col-sm-8">The rankscore is the ratio of the rank of the score over the total number of
                    MetaSVM scores in dbNSFP. The scores range from 0 to 1.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">MetaSVM_pred</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Prediction of our SVM based ensemble prediction score,"T(olerated)" or</dd>
                <dd class="col-sm-8">"D(amaging)". The score cutoff between "D" and "T" is 0. The rankscore cutoff
                    between "D" and "T" is 0.82257.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">MetaLR_score</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Our logistic regression (LR) based ensemble prediction score, which</dd>
                <dd class="col-sm-8">incorporated 10 scores (SIFT, PolyPhen-2 HDIV, PolyPhen-2 HVAR, GERP++,
                    MutationTaster, Mutation Assessor, FATHMM, LRT, SiPhy, PhyloP) and the maximum frequency observed in
                    the 1000 genomes populations. Larger value means the SNV is more likely to be damaging. Scores range
                    from 0 to 1.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">MetaLR_rankscore</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">MetaLR scores were ranked among all MetaLR scores in dbNSFP. The rankscore</dd>
                <dd class="col-sm-8">is the ratio of the rank of the score over the total number of MetaLR scores in
                    dbNSFP. The scores range from 0 to 1.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">MetaLR_pred</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Prediction of our MetaLR based ensemble prediction score,"T(olerated)" or</dd>
                <dd class="col-sm-8">"D(amaging)". The score cutoff between "D" and "T" is 0.5. The rankscore cutoff
                    between "D" and "T" is 0.81101.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">Reliability_index</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Number of observed component scores (except the maximum frequency in</dd>
                <dd class="col-sm-8">the 1000 genomes populations) for MetaSVM and MetaLR. Ranges from 1 to 10. As
                    MetaSVM and MetaLR scores are calculated based on imputed data, the less missing component scores,
                    the higher the reliability of the scores and predictions.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">M-CAP_score</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">M-CAP score (details in DOI</dd>
                <dd class="col-sm-8">10.1038/ng.3703). Scores range from 0 to 1. The larger the score the more likely
                    the SNP has damaging effect.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">M-CAP_rankscore</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">M-CAP scores were ranked among all M-CAP scores in dbNSFP. The rankscore is</dd>
                <dd class="col-sm-8">the ratio of the rank of the score over the total number of M-CAP scores in dbNSFP.
                </dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">M-CAP_pred</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Prediction of M-CAP score based on the authors' recommendation, "T(olerated)" or
                </dd>
                <dd class="col-sm-8">"D(amaging)". The score cutoff between "D" and "T" is 0.025.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">MutPred_score</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">General MutPred score. Scores range from 0 to 1. The larger the score the more</dd>
                <dd class="col-sm-8">likely the SNP has damaging effect.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">MutPred_rankscore</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">MutPred scores were ranked among all MutPred scores in dbNSFP. The rankscore is
                </dd>
                <dd class="col-sm-8">the ratio of the rank of the score over the total number of MutPred scores in
                    dbNSFP.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">MutPred_protID</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">UniProt accession or Ensembl transcript ID used for MutPred_score calculation.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">MutPred_AAchange</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Amino acid change used for MutPred_score calculation.</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">MutPred_Top5features</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Top 5 features (molecular mechanisms of disease) as predicted by MutPred with</dd>
                <dd class="col-sm-8">p values. MutPred_score > 0.5 and p < 0.05 are referred to as actionable
                        hypotheses. MutPred_score> 0.75 and p < 0.05 are referred to as confident hypotheses.
                            MutPred_score> 0.75 and p < 0.01 are referred to as very confident hypotheses.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">MVP_score</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">A pathogenicity prediction score for missense variants using deep learning
                    approach.</dd>
                <dd class="col-sm-8">The range of MVP score is from 0 to 1. The larger the score, the more likely the
                    variant is pathogenic. The authors suggest thresholds of 0.7 and 0.75 for separating damaging vs
                    tolerant variants in constrained genes (ExAC pLI >=0.5) and non-constrained genes (ExAC pLI<0.5),
                        respectively. Details see doi http://dx.doi.org/10.1101/259390 Multiple entries are separated
                        by ";" , corresponding to Ensembl_transcriptid.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">MVP_rankscore</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">MVP scores were ranked among all MVP scores in dbNSFP. The rankscore is</dd>
                <dd class="col-sm-8">the ratio of the rank of the score over the total number of MVP scores in dbNSFP.
                </dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">MPC_score</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">A deleteriousness prediction score for missense variants based on regional missense
                </dd>
                <dd class="col-sm-8">constraint. The range of MPC score is 0 to 5. The larger the score, the more likely
                    the variant is pathogenic. Details see doi http://dx.doi.org/10.1101/148353. Multiple entries are
                    separated by ";", corresponding to Ensembl_transcriptid.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">MPC_rankscore</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">MPC scores were ranked among all MPC scores in dbNSFP. The rankscore is</dd>
                <dd class="col-sm-8">the ratio of the rank of the score over the total number of MPC scores in dbNSFP.
                </dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">PrimateAI_score</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">A pathogenicity prediction score for missense variants based on common variants of
                </dd>
                <dd class="col-sm-8">non-human primate species using a deep neural network. The range of PrimateAI score
                    is 0 to 1. The larger the score, the more likely the variant is pathogenic. The authors suggest a
                    threshold of 0.803 for separating damaging vs tolerant variants. Details see
                    https://doi.org/10.1038/s41588-018-0167-z</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">PrimateAI_rankscore</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">PrimateAI scores were ranked among all PrimateAI scores in dbNSFP. The rankscore is
                </dd>
                <dd class="col-sm-8">the ratio of the rank of the score over the total number of PrimateAI scores in
                    dbNSFP.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">PrimateAI_pred</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Prediction of PrimateAI score based on the authors' recommendation, "T(olerated)"
                    or</dd>
                <dd class="col-sm-8">"D(amaging)". The score cutoff between "D" and "T" is 0.803.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">DEOGEN2_score</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">A deleteriousness prediction score "which incorporates heterogeneous information
                    about</dd>
                <dd class="col-sm-8">the molecular effects of the variants, the domains involved, the relevance of the
                    gene and the interactions in which it participates". It ranges from 0 to 1. The larger the score,
                    the more likely the variant is deleterious. The authors suggest a threshold of 0.5 for separating
                    damaging vs tolerant variants.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">DEOGEN2_rankscore</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">DEOGEN2 scores were ranked among all DEOGEN2 scores in dbNSFP. The rankscore is
                </dd>
                <dd class="col-sm-8">the ratio of the rank of the score over the total number of DEOGEN2 scores in
                    dbNSFP.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">DEOGEN2_pred</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Prediction of DEOGEN2 score based on the authors' recommendation, "T(olerated)" or
                </dd>
                <dd class="col-sm-8">"D(amaging)". The score cutoff between "D" and "T" is 0.5.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">Aloft_Fraction_transcripts_affected</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">the fraction of the transcripts of the gene affected</dd>
                <dd class="col-sm-8">i.e. No. of transcripts affected by the SNP/Total no. of protein_coding transcripts
                    for the gene multiple values separated by ";", corresponding to Ensembl_proteinid.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">Aloft_prob_Tolerant</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Probability of the SNP being classified as benign by ALoFT</dd>
                <dd class="col-sm-8">multiple values separated by ";", corresponding to Ensembl_proteinid.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">Aloft_prob_Recessive</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Probability of the SNP being classified as recessive disease-causing by ALoFT</dd>
                <dd class="col-sm-8">multiple values separated by ";", corresponding to Ensembl_proteinid.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">Aloft_prob_Dominant</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> Probability of the SNP being classified as dominant disease-causing by ALoFT</dd>
                <dd class="col-sm-8">multiple values separated by ";", corresponding to Ensembl_proteinid.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">Aloft_pred</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">final classification predicted by ALoFT;</dd>
                <dd class="col-sm-8">values can be Tolerant, Recessive or Dominant multiple values separated by ";",
                    corresponding to Ensembl_proteinid.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">Aloft_Confidence</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4">Confidence level of Aloft_pred;</dd>
                <dd class="col-sm-8">values can be "High Confidence" (p < 0.05) or "Low Confidence" (p> 0.05) multiple
                        values separated by ";", corresponding to Ensembl_proteinid.</dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">PANTHER Functional Annotation based on Different Flanking Region:</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"></dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_0_GO_molecular_function_complete_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_0_GO_biological_process_complete_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_0_GO_biological_process_complete_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_0_GO_cellular_component_complete_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_0_GO_cellular_component_complete_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_0_PANTHER_GO_SLIM_molecular_function_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_0_PANTHER_GO_SLIM_molecular_function_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_0_PANTHER_GO_SLIM_biological_process_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_0_PANTHER_GO_SLIM_biological_process_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_0_PANTHER_GO_SLIM_cellular_component_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_0_PANTHER_GO_SLIM_cellular_component_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_0_REACTOME_pathway_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_0_REACTOME_pathway_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_0_PANTHER_pathway_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_0_PANTHER_pathway_lis_idt</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_10000_GO_molecular_function_complete_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_10000_GO_molecular_function_complete_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_10000_GO_biological_process_complete_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_10000_GO_biological_process_complete_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_10000_GO_cellular_component_complete_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_10000_GO_cellular_component_complete_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_10000_PANTHER_GO_SLIM_molecular_function_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_10000_PANTHER_GO_SLIM_molecular_function_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_10000_PANTHER_GO_SLIM_biological_process_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_10000_PANTHER_GO_SLIM_biological_process_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_10000_PANTHER_GO_SLIM_cellular_component_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_10000_PANTHER_GO_SLIM_cellular_component_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_10000_REACTOME_pathway_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_10000_REACTOME_pathway_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_10000_PANTHER_pathway_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_10000_PANTHER_pathway_lis_idt</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_20000_GO_molecular_function_complete_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_20000_GO_molecular_function_complete_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_20000_GO_biological_process_complete_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_20000_GO_biological_process_complete_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_20000_GO_cellular_component_complete_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_20000_GO_cellular_component_complete_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_20000_PANTHER_GO_SLIM_molecular_function_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_20000_PANTHER_GO_SLIM_molecular_function_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_20000_PANTHER_GO_SLIM_biological_process_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_20000_PANTHER_GO_SLIM_biological_process_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_20000_PANTHER_GO_SLIM_cellular_component_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_20000_PANTHER_GO_SLIM_cellular_component_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_20000_REACTOME_pathway_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_20000_REACTOME_pathway_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_20000_PANTHER_pathway_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> flanking_20000_PANTHER_pathway_lis_idt</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">Functional Annotations linked by Annotation Tools Result:</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"></dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_ensembl_GO_molecular_function_complete_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_ensembl_GO_molecular_function_complete_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_ensembl_GO_biological_process_complete_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_ensembl_GO_biological_process_complete_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_ensembl_GO_cellular_component_complete_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_ensembl_GO_cellular_component_complete_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_ensembl_PANTHER_GO_SLIM_molecular_function_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_ensembl_PANTHER_GO_SLIM_molecular_function_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_ensembl_PANTHER_GO_SLIM_biological_process_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_ensembl_PANTHER_GO_SLIM_biological_process_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_ensembl_PANTHER_GO_SLIM_cellular_component_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_ensembl_PANTHER_GO_SLIM_cellular_component_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_ensembl_REACTOME_pathway_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_ensembl_REACTOME_pathway_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_ensembl_PANTHER_pathway_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_ensembl_PANTHER_pathway_lis_idt</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_refseq_GO_molecular_function_complete_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_refseq_GO_molecular_function_complete_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_refseq_GO_biological_process_complete_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_refseq_GO_biological_process_complete_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_refseq_GO_cellular_component_complete_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_refseq_GO_cellular_component_complete_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_refseq_PANTHER_GO_SLIM_molecular_function_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_refseq_PANTHER_GO_SLIM_molecular_function_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_refseq_PANTHER_GO_SLIM_biological_process_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_refseq_PANTHER_GO_SLIM_biological_process_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_refseq_PANTHER_GO_SLIM_cellular_component_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_refseq_PANTHER_GO_SLIM_cellular_component_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_refseq_REACTOME_pathway_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_refseq_REACTOME_pathway_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_refseq_PANTHER_pathway_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> ANNOVAR_refseq_PANTHER_pathway_lis_idt</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_ensembl_GO_molecular_function_complete_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_ensembl_GO_molecular_function_complete_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_ensembl_GO_biological_process_complete_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_ensembl_GO_biological_process_complete_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_ensembl_GO_cellular_component_complete_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_ensembl_GO_cellular_component_complete_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_ensembl_PANTHER_GO_SLIM_molecular_function_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_ensembl_PANTHER_GO_SLIM_molecular_function_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_ensembl_PANTHER_GO_SLIM_biological_process_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_ensembl_PANTHER_GO_SLIM_biological_process_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_ensembl_PANTHER_GO_SLIM_cellular_component_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_ensembl_PANTHER_GO_SLIM_cellular_component_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_ensembl_REACTOME_pathway_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_ensembl_REACTOME_pathway_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_ensembl_PANTHER_pathway_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_ensembl_PANTHER_pathway_lis_idt</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_refseq_GO_molecular_function_complete_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_refseq_GO_molecular_function_complete_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_refseq_GO_biological_process_complete_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_refseq_GO_biological_process_complete_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_refseq_GO_cellular_component_complete_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_refseq_GO_cellular_component_complete_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_refseq_PANTHER_GO_SLIM_molecular_function_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_refseq_PANTHER_GO_SLIM_molecular_function_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_refseq_PANTHER_GO_SLIM_biological_process_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_refseq_PANTHER_GO_SLIM_biological_process_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_refseq_PANTHER_GO_SLIM_cellular_component_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_refseq_PANTHER_GO_SLIM_cellular_component_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_refseq_REACTOME_pathway_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_refseq_REACTOME_pathway_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_refseq_PANTHER_pathway_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> SnpEff_refseq_PANTHER_pathway_lis_idt</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_ensembl_GO_molecular_function_complete_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_ensembl_GO_molecular_function_complete_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_ensembl_GO_biological_process_complete_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_ensembl_GO_biological_process_complete_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_ensembl_GO_cellular_component_complete_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_ensembl_GO_cellular_component_complete_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_ensembl_PANTHER_GO_SLIM_molecular_function_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_ensembl_PANTHER_GO_SLIM_molecular_function_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_ensembl_PANTHER_GO_SLIM_biological_process_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_ensembl_PANTHER_GO_SLIM_biological_process_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_ensembl_PANTHER_GO_SLIM_cellular_component_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_ensembl_PANTHER_GO_SLIM_cellular_component_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_ensembl_REACTOME_pathway_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_ensembl_REACTOME_pathway_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_ensembl_PANTHER_pathway_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_ensembl_PANTHER_pathway_lis_idt</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_refseq_GO_molecular_function_complete_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_refseq_GO_molecular_function_complete_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_refseq_GO_biological_process_complete_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_refseq_GO_biological_process_complete_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_refseq_GO_cellular_component_complete_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_refseq_GO_cellular_component_complete_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_refseq_PANTHER_GO_SLIM_molecular_function_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_refseq_PANTHER_GO_SLIM_molecular_function_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_refseq_PANTHER_GO_SLIM_biological_process_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_refseq_PANTHER_GO_SLIM_biological_process_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_refseq_PANTHER_GO_SLIM_cellular_component_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_refseq_PANTHER_GO_SLIM_cellular_component_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_refseq_REACTOME_pathway_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_refseq_REACTOME_pathway_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_refseq_PANTHER_pathway_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> VEP_refseq_PANTHER_pathway_lis_idt</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4">IMAGE Enhancer Functional Annotations:</dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"></dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> enhancer_linked_genes</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> enhancer_linked_assay</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> enhancer_linked_tissue</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> enhancer_linked_GO_molecular_function_complete_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> enhancer_linked_GO_molecular_function_complete_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> enhancer_linked_GO_biological_process_complete_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> enhancer_linked_GO_biological_process_complete_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> enhancer_linked_GO_cellular_component_complete_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> enhancer_linked_GO_cellular_component_complete_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> enhancer_linked_PANTHER_GO_SLIM_molecular_function_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> enhancer_linked_PANTHER_GO_SLIM_molecular_function_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> enhancer_linked_PANTHER_GO_SLIM_biological_process_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> enhancer_linked_PANTHER_GO_SLIM_biological_process_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> enhancer_linked_PANTHER_GO_SLIM_cellular_component_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> enhancer_linked_PANTHER_GO_SLIM_cellular_component_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> enhancer_linked_REACTOME_pathway_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> enhancer_linked_REACTOME_pathway_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> enhancer_linked_PANTHER_pathway_list</dd>
                <dd class="col-sm-8"></dd>
            </dl>
            <dl class="row">
                <dt class="col-sm-4"></dt>
            </dl>
            <dl class="row">
                <dd class="col-sm-4"> enhancer_linked_PANTHER_pathway_list_id</dd>
                <dd class="col-sm-8"></dd>
            </dl>

        </div>
    </div>


    <!-- /Content -->

    <!-- Footer -->
    <div class="footer gradient-2">
        <div class="container footer-container ">
            <div class="row">
                <div class="col-xs-4 col-sm-3 col-md-3 col-lg-3">
                    <div class="footer-title"><img src="/static/main/img/image_logo.gif" style="height:80px"
                            alt="USC IMAGE logo"></div>
                    <ul class="list-unstyled">
                    </ul>
                </div>
                <div class="col-xs-4 col-sm-3 col-md-3 col-lg-3">

                </div>
                <div class="col-xs-4 col-sm-3 col-md-3 col-lg-3">
                    <div class="footer-title"></div>
                    <ul class="list-unstyled">
                        <li><a href="http://annoq.org:4206/" target="_blank">Interactive Query</a></li>
                        <li><a href="https://uscbiostats.github.io/AnnoQ/api" target="_blank">Data Access</a></li>
                        <li><a href="/detail.html" target="_blank">Annotation Document</a></li>
                        <li><a href="https://uscbiostats.github.io/AnnoQ/" target="_blank">About</a></li>
                    </ul>
                </div>
                <div class="col-xs-12 col-sm-3 col-md-3 col-lg-3">
                </div>
            </div>
        </div>
    </div>