<div class="w-100-p" noctuaPerfectScrollbar>

    <div class="header accent p-8 w-100-p">
        <div class="annoq-container" fxLayout="row" fxLayoutAlign="start center">
            <div fxLayout="column" fxLayoutAlign="center center">
                <h1 [@animate]="{value:'*',params:{delay:'100ms', y:'25px'}}">
                    UI Tutorial
                </h1>
            </div>
        </div>
    </div>
    <div class="annoq-section">
        <div class="annoq-container" fxLayout="column" fxLayoutAlign="start start">
            <div class="annoq-image" fxFlex="60%">
                <img class="" src="assets/images/short.gif">
            </div>
            <h1>Getting Started Tutorial</h1>
            <p>The UI is composed of 3 main panels, the Query Panel, Result Panel and Result Summary Panel</p>
            <h2>1. Queries (Query Panel)</h2>
            <p>There 5 types of supported queries as described below (1-5). Choose a query type from the dropdown
                labeled
                “Change”.</p>
            <h3>1.1 Choose Query Type</h3>
            <p>Genome Coordinates Query: Chromosome location. Specify the chromosome number and the range. All data
                is
                based on hg19.
                Query by Upload VCF file: Variants List. Upload a VCF file of up to 10,000 variants by clicking the
                "Populate from a File" button. A sample VCF file can be found at here. Note that only first 50 rows
                of
                result will be displayed by our website. For retrieving the full result please using the download
                function.
                Gene Product Query: Gene Product. Input a gene or protein ID (e.g., ZMYND11). All variants located
                in
                the
                gene region will be returned. Currently, the system can only query one gene at a time. Gene name
                will
                be
                mapped to HGNC gene id by PANTHER gene mapping API.
                rsID query: Input a single rs ID and retrieve annotations.
                General Keyword Search: AnnoQ also supports full-text search by using a keyword. This keyword can be
                a
                gene
                name, a phenotype name or a GO term (e.g. Signaling by GPCR).</p>
            <h3>1.2 Select annotations</h3>
            <p>The annotations are organized in tree structure of categories and its individual annotations.</p>
            <p>Select the entire category by clicking on the checkbox (such as ANNOVAR).</p>
            <p>To choose individual annotations in a category, expand the tree by clicking the triangle next to the
                checkbox and click the desired checkboxes.</p>
            <h3>1.3 Submit query</h3>
            <p>Submit query by clicking the “Submit” button in the bottom of the panel. The results will be
                displayed on
                the result panel of the page.</p>
            <h3>1.4 Export configuration file</h3>
            <p>There are over 400 annotation types stored in our database. Users may not need all of them in their
                analysis. Through the process above, users can view the results and decide which annotation to use
                in
                their
                analysis. The configuration file stores the annotation selections the user chooses (from B above),
                and
                is
                used for command-line query or embedded in a programming script such as R. To generate a
                configuration
                file, simply click the “Export Config” button, and follow the instruction to save the file on the
                computer.
                For instructions how to use the configuration file in the command-line query or programming scripts,
                please
                visit link and AnnoQR. </p>
            <h2>2. Query Results (Results Panel)</h2>
            <p>The results are represented in a table format where each column ….</p>
            <p>To select an individual row for detailed result, click the row and the summary panel will appear</p>
            <h3>2.1 Download results</h3>
            <p>The results displayed on the results panel can be downloaded. To do so, click the “Download” button
                on the
                right upper corner of the results panel. The downloaded file is generated on the server. A link will
                be
                displayed. The user can click the link to download the file.</p>
            <p>It may take some time to generate the file for downloading.</p>
            <h2>3. Summary Panel</h2>
            <p>Select a row to see it in a detailed summary</p>
        </div>
    </div>
    <annoq-footer></annoq-footer>
</div>